blenderset/background.py

```diff
@@ -292,12 +292,15 @@ def __init__(
         self, context, blend_file, camera_name=None, apply_background_modifiers=True
     ):
         super().__init__(context)
-        self.blend_file = str(self.root / "background_models" / blend_file)
+        if isinstance(blend_file, str):
```

Comment:
Please add short documentation for intended use of blend_files, whether root should be included or not depends on if it is a string or list.

Reply:
Thank you for the documentation.
If I remember correctly, the isinstance(blend_file, str) is to be backwards compatible and should not be used?
From the lines below it looks like the blend_file list should actually contain the entire path including self.root / "background_models" or wherever the files are located? If so we would recommend updating "filenames" to "file paths" in the documentation above. 

(also some typos: "whoes" -> "whos" and "backgournd" -> "background")

As it is clear that blend_file should be a list from the description, perhaps we can rename the variable to blend_files? Or does that break the backwards compatibility for you?

Reply:
Good idea! Should be fixed now.